e2e-tests/pipeline_validation.sh

```diff
@@ -112,12 +114,20 @@ function setup() {
   SOURCE_FHIR_SERVER_URL='http://localhost:8091'
   STREAMING=""
   OPENMRS=""
+  # This global variable is hardcoded to validate the View record count
+  # which can greater than the number of Resources in the source FHIR
+  # Server due to flattening
+  PARQUET_VIEW_ROWCOUNT=0
```

Comment:
These can be local variables in the method test_parquet_sink() instead of global variables and can initialized in the same method.

Reply:
Done.